target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-wtr-m2133hp.dts

```diff
@@ -0,0 +1,420 @@
+	status = "okay";
+};
+
+&cryptobam {
+	status = "okay";
+};
+
+&pcie0 {
+	status = "okay";
+
+	bridge@0,0 {
+		reg = <0x00000000 0 0 0 0>;
+		#address-cells = <3>;
+		#size-cells = <2>;
+		ranges;
+
+		wifi@0,0 {
+			compatible = "qcom,ath10k";
+			reg = <0 0 0 0 0>;
+
+			qcom,ath10k-calibration-variant = "Buffalo-WTR-M2133HP";
+		};
+	};
+};
+
+&qpic_bam {
+	status = "okay";
+};
+
+&nand {
+	pinctrl-0 = <&nand_pins>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	nand@0 {
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "SBL1";
+				reg = <0x0000000 0x0100000>;
+				read-only;
+			};
+
+			partition@100000 {
+				label = "MIBIB";
+				reg = <0x0100000 0x0100000>;
+				read-only;
+			};
+
+			partition@200000 {
+				label = "BOOTCONFIG";
+				reg = <0x0200000 0x0100000>;
+				read-only;
+			};
+
+			partition@300000 {
+				label = "QSEE";
+				reg = <0x0300000 0x0100000>;
+				read-only;
+			};
+
+			partition@400000 {
+				label = "QSEE_1";
+				reg = <0x0400000 0x0100000>;
+				read-only;
+			};
+
+			partition@500000 {
+				label = "CDT";
+				reg = <0x0500000 0x0080000>;
+				read-only;
+			};
+
+			partition@580000 {
+				label = "CDT_1";
+				reg = <0x0580000 0x0080000>;
+				read-only;
+			};
+
+			partition@600000 {
+				label = "BOOTCONFIG1";
+				reg = <0x0600000 0x0080000>;
+				read-only;
+			};
+
+			partition@680000 {
+				label = "APPSBLENV";
+				reg = <0x0680000 0x0080000>;
+			};
+
+			partition@700000 {
+				label = "APPSBL";
+				reg = <0x0700000 0x0200000>;
+				read-only;
+			};
+
+			partition@900000 {
+				label = "APPSBL_1";
+				reg = <0x0900000 0x0200000>;
+				read-only;
+			};
+
+			art: partition@b00000 {
+				label = "ART";
+				reg = <0x0b00000 0x0080000>;
+				read-only;
+			};
+
+			partition@b80000 {
+				label = "ART_1";
+				reg = <0x0b80000 0x0080000>;
+				read-only;
+			};
+
+			orgdata: partition@c00000 {
+				label = "ORGDATA";
+				reg = <0x0c00000 0x0080000>;
+				read-only;
+			};
+
+			partition@c80000 {
+				label = "ORGDATA_1";
+				reg = <0x0c80000 0x0080000>;
+				read-only;
+			};
+
+			partition@d00000 {
+				label = "ubi";
+				reg = <0x0d00000 0x2900000>;
+			};
+
+			partition@3600000 {
+				label = "rootfs_recover";
+				reg = <0x3600000 0x2900000>;
+				read-only;
+			};
+
+			partition@5f00000 {
+				label = "user_property";
+				reg = <0x5f00000 0x1a20000>;
+				read-only;
+			};
+		};
+	};
+};
+
+&wifi0 {
+	status = "okay";
+	qcom,ath10k-calibration-variant = "Buffalo-WTR-M2133HP";
+	ieee80211-freq-limit = <2400000 2483000>;
+};
+
+&wifi1 {
+	status = "okay";
+	qcom,ath10k-calibration-variant = "Buffalo-WTR-M2133HP";
+};
+
+&mdio {
+	status = "okay";
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+	reset-gpios = <&tlmm 47 GPIO_ACTIVE_LOW>;
+};
+
+&gmac0 {
+	mtd-mac-address = <&orgdata 0x20>;
+};
+
+&gmac1 {
+	mtd-mac-address = <&orgdata 0x20>;
+	mtd-mac-address-increment = <1>;
```

Comment:
So, this is the only interface without a hard-coded address?

Reply:
In vendor firmware, eth0 and eth1 uses the same MAC Address (&orgdata 0x20).

Reply:
Then OpenWrt should do the same.

Reply:
Okay. Fixed.